code/modules/pai/pai.dm

```diff
@@ -37,6 +37,8 @@
 	var/can_transmit = TRUE
 	/// The card we inhabit
 	var/obj/item/pai_card/card
+	/// The maximum distance we can travel away from our pai card
+	var/max_distance = 5
```

Comment:
```suggestion
	var/leashed_distance = 5
```

makes more sense as to not be confused with `range_max` and `range_min` and whatever else- best to not co-mingle quantity-based names.